articles/azure-functions/analyze-telemetry-data.md

```diff
@@ -103,6 +103,35 @@ traces
 
 The runtime provides the `customDimensions.LogLevel` and `customDimensions.Category` fields. You can provide additional fields in logs that you write in your function code. For an example in C#, see [Structured logging](functions-dotnet-class-library.md#structured-logging) in the .NET class library developer guide.
 
+## Query Function Invocation
+
+Every Function invocation is assigned a unique id. InvocationId is included in the custom dimention and can be used to coorelate all the logs from a particular function invocation.
```

Comment:
Just some minor edits. Note that Azure docs use `Sentence case` and not `Title Case` for headings. 
```suggestion
## Query function invocations

Every function invocation is assigned a unique ID. `InvocationId` is included in the custom dimension and can be used to correlate all the logs from a particular function execution.
```